src/discord-refcount.c

```diff
@@ -117,40 +119,83 @@ _discord_refcounter_contains(struct discord_refcounter *rc, const void *data)
     return ret;
 }
 
-bool
+static CCORDcode
+_discord_refcounter_incr_no_lock(struct discord_refcounter *rc, void *data)
+{
+    CCORDcode code = CCORD_UNAVAILABLE;
+    if (_discord_refcounter_contains(rc, data)) {
+        struct _discord_refvalue *value = _discord_refvalue_find(rc, data);
+
+        ++value->visits;
```

Comment:
There should probably be an error if (value->visits == MAX_INT)

Reply:
Done.